ocean_lib/assets/v4/test/test_asset.py

```diff
@@ -92,7 +92,10 @@ def test_asset_utils(web3):
     }
     event = ddo_dict["event"]
 
-    assert ddo_dict["stats"] == {"consumes": 4, "isInPurgatory": "false"}
+    assert ddo_dict["purgatory"] == {"state": False}
+    purgatory = ddo_dict["purgatory"]
+
+    assert ddo_dict["stats"] == {"consumes": 4}
```

Comment:
how did "4" happen here?

Reply:
It's a sample asset, it's just for testing.